README.asciidoc

```diff
@@ -162,12 +162,16 @@ To configure default method serialization, simply configure the `@Parcel` annota
 public class Example {
     private String name;
     private int age;
+    private boolean enable;
```

Comment:
Can you rename this property and the associated getters and settes to "enabled" so the getter can read "is-enabled"?